airbyte-integrations/connectors/source-sftp-bulk/metadata.yaml

```diff
@@ -1,14 +1,14 @@
 data:
   resourceRequirements:
     jobSpecific:
-    - jobType: check_connection
```

Comment:
I started from master where the formatting was wrong. You can ignore this as it'll disappear once I merge master into this